src/tyr/serializers.cc

```diff
@@ -83,10 +85,48 @@ std::vector<std::string> openlr_edges(const TripLeg& leg) {
 } // namespace
 namespace valhalla {
 namespace tyr {
-std::string serializeStatus(const Api&) {
-  // TODO: once we decide on what's in the status message we'll fill out the proto message in
-  // loki/thor/odin and we'll serialize it here
-  return "{}";
+std::string serializeStatus(const Api& request) {
+
+  rapidjson::Document status_doc;
+  status_doc.SetObject();
+  auto& alloc = status_doc.GetAllocator();
+
+  if (request.status().has_version())
+    status_doc.AddMember("version", rapidjson::Value().SetString(request.status().version(), alloc),
+                         alloc);
+  if (request.status().has_has_tiles())
+    status_doc.AddMember("has_tiles", rapidjson::Value().SetBool(request.status().has_tiles()),
+                         alloc);
+  if (request.status().has_has_admins())
+    status_doc.AddMember("has_admins", rapidjson::Value().SetBool(request.status().has_admins()),
+                         alloc);
+  if (request.status().has_has_timezones())
+    status_doc.AddMember("has_timezones",
+                         rapidjson::Value().SetBool(request.status().has_timezones()), alloc);
+  if (request.status().has_has_live_traffic())
+    status_doc.AddMember("has_live_traffic",
+                         rapidjson::Value().SetBool(request.status().has_live_traffic()), alloc);
+
+  rapidjson::Document bbox_doc;
+  if (request.status().has_bbox()) {
+    bbox_doc.Parse(request.status().bbox());
+    rapidjson::SetValueByPointer(status_doc, "/bbox", bbox_doc, alloc);
+  }
+
+  // add the entire config and remove sensible paths
+  rapidjson::Document config_doc;
+  if (request.status().has_config()) {
+    config_doc.Parse(request.status().config());
+    for (const auto& path :
+         {"/mjolnir/tile_dir", "/mjolnir/tile_extract", "/mjolnir/admin", "/mjolnir/timezone",
+          "/mjolnir/transit_dir", "/additional_data/elevation"}) {
+      config_doc.RemoveMember(path);
+    }
+    status_doc.AddMember("config", config_doc, alloc);
+  }
+
+  // if not verbose, return empty string
+  return status_doc.ObjectEmpty() ? "" : rapidjson::to_string(status_doc);
```

Comment:
return empty string rather than empty object

Reply:
but why? all of our apis are json. i guess you are thinking what is the point of an empty object. my only rebuttal is that then the client can at least parse every single response as json and not have one special condition that changes the response format `""` isnt valid json

Reply:
good point